pyproject.toml

```diff
@@ -136,7 +136,11 @@ test-command = [
 profile = "black"
 
 [tool.mypy]
+disallow_untyped_defs = true
+exclude = "^((tests|docs|examples|bin)/|conftest.py|src/clib/|src/xtgeo/(common|cube|xtgeo|grid3d|metadata|plot|roxutils|surface|well|xyz)/?)"
+follow_imports = "skip"
```

Comment:
For now, lets focus on our own code. There are loads of "stubs" we can add that has type-annotations for third parties, but I think we should deal with that in a separate pr.